Intersect.Client/Entities/Player.cs

```diff
@@ -1534,57 +1549,104 @@ public bool TryBlock()
 
         public bool TryAttack()
         {
-            if (AttackTimer > Timing.Global.Milliseconds || IsBlocking || (IsMoving && !Options.Instance.PlayerOpts.AllowCombatMovement))
+            if (AttackTimer > Timing.Global.Milliseconds || IsBlocking ||
+                (IsMoving && !Options.Instance.PlayerOpts.AllowCombatMovement))
```

Comment:
Honestly, as much as it sucks to have multiple if-statements there is 0 performance downside to it, please separate these 3 comparisons into multiple if statements, starting with IsBlocking, then AttackTimer, and then IsMoving && !AllowCombatMovement.

Reply:
Done 👍🏽